src/redux/categories/Categories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <h1>Under construction</h1>
```

Comment:
**Changes Required**
- [ ] The text `Under Construction` not being displayed. The current navbar is blocking it. I would advice that you don't put much effort on the styling for now. This is a section for styling later.

Reply:
well done